addons/host-upgrades/resources/40-configmap.yml

```diff
@@ -0,0 +1,159 @@
+    # minimal                            = yum --bugfix update-minimal
+    # minimal-security                   = yum --security update-minimal
+    # minimal-security-severity:Critical =  --sec-severity=Critical update-minimal
+    update_cmd = default
+
+    # Whether a message should be emitted when updates are available,
+    # were downloaded, or applied.
+    update_messages = yes
+
+    # Whether updates should be downloaded when they are available.
+    download_updates = yes
+
+    # Whether updates should be applied when they are available.  Note
+    # that download_updates must also be yes for the update to be applied.
+    apply_updates = yes
+
+    # Maximum amout of time to randomly sleep, in minutes.  The program
+    # will sleep for a random amount of time between 0 and random_sleep
+    # minutes before running.  This is useful for e.g. staggering the
+    # times that multiple systems will access update servers.  If
+    # random_sleep is 0 or negative, the program will run immediately.
+    # 6*60 = 360
+    random_sleep = 0
+
+    [emitters]
+    # Name to use for this system in messages that are emitted.  If
+    # system_name is None, the hostname will be used.
+    system_name = None
+
+    # How to send messages.  Valid options are stdio and email.  If
+    # emit_via includes stdio, messages will be sent to stdout; this is useful
+    # to have cron send the messages.  If emit_via includes email, this
+    # program will send email itself according to the configured options.
+    # If emit_via is None or left blank, no messages will be sent.
+    emit_via = stdio
+
+    # The width, in characters, that messages that are emitted should be
+    # formatted to.
+    output_width = 80
+
+
+    [email]
+    # The address to send email messages from.
+    # NOTE: 'localhost' will be replaced with the value of system_name.
+    email_from = root@localhost
+
+    # List of addresses to send messages to.
+    email_to = root
+
+    # Name of the host to connect to to send email messages.
+    email_host = localhost
+
+
+    [groups]
+    # NOTE: This only works when group_command != objects, which is now the default
+    # List of groups to update
+    group_list = None
+
+    # The types of group packages to install
+    group_package_types = mandatory, default
+
+    [base]
+    # This section overrides yum.conf
+
+    # Use this to filter Yum core messages
+    # -4: critical
+    # -3: critical+errors
+    # -2: critical+errors+warnings (default)
+    debuglevel = 1
+
+    # skip_broken = True
+    mdpolicy = group:main
+
+    # Uncomment to auto-import new gpg keys (dangerous)
+    # assumeyes = True
+
+  unattended-upgrades.conf: |
+    // Override system /etc/apt/apt.conf.d/50unattended-upgrades
+    #clear "Unattended-Upgrade::Allowed-Origins";
+
+    // Automatically upgrade packages from these (origin:archive) pairs
+    Unattended-Upgrade::Allowed-Origins {
+    // "${distro_id}:${distro_codename}";
```

Comment:
The `Ubuntu:xenial` origin is whitelisted in the default configuration, AFAIK that corresponds to LTS point releases (16.04.x):

```
// Automatically upgrade packages from these (origin:archive) pairs
Unattended-Upgrade::Allowed-Origins {
	"${distro_id}:${distro_codename}";
	"${distro_id}:${distro_codename}-security";
	// Extended Security Maintenance; doesn't necessarily exist for
	// every release and this system may not have it installed, but if
	// available, the policy for updates is such that unattended-upgrades
	// should also install from here by default.
	"${distro_id}ESM:${distro_codename}";
//	"${distro_id}:${distro_codename}-updates";
//	"${distro_id}:${distro_codename}-proposed";
//	"${distro_id}:${distro_codename}-backports";
};
```